### DISCUSSION TITLE
Problems with Registration

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#941] [Help] [answered]
Upon Authorizing Vercel into my Github i get This Error: `This user account is blocked. Contact registration@vercel.com for more information.`

I can see Vercel in my Authorized Github Apps but that doesn't do anything in this case apparently.

Continuing with E-Mail i get Another Error: `Sorry, we are unable to validate that email. Please contact us at [registration@vercel.com](mailto:registration@vercel.com) for further assistance.`

**Comment by swarnava** [comment, marked as answer]
As the error message suggest, please reach out to registration@vercel.com to review your account.
